meteor/sofie-server/model/exceptions/unsupported-operation.ts

```diff
@@ -0,0 +1,8 @@
+import { Exception } from './exception'
+import { ErrorCode } from '../enums/error-code'
+
+export class UnsupportedOperation extends Exception {
+	constructor(message?: string) {
```

Comment:
When would we not give a message?

Reply:
When we are lazy? ;)
I don't know, I will remove the optional

Reply:
I also just now removed the optional on the two exception i have created for my task.

Could it make sense to remove it all or most of the exceptions?
Also is the `RundownIsLockedException` still needed?

Reply:
> Also is the RundownIsLockedException still needed?

Currently no, so it should be deleted. When we implement the "locking" mechanism, then we can see if it make sense to bring it back.

Good catch. I will delete it :)